dependencies.yaml

```
@@ -145,6 +159,26 @@ dependencies:
       match: REVISION:\ '\d+'
 
   # kube-cross
+  - name: "registry.k8s.io/build-image/kube-cross (v1.20-go1.22)"
```

Shouldn't this be v1.30?
```suggestion
  - name: "registry.k8s.io/build-image/kube-cross (v1.30-go1.22)"
```